library/core/src/unicode/unicode_data.rs

```diff
@@ -316,6 +316,7 @@ pub mod grapheme_extend {
         128, 240, 0,
     ];
     pub fn lookup(c: char) -> bool {
+        (c as u32) >= 768 &&
```

Comment:
I haven’t really spent much time figuring out how the skip list search actually works, and what the meaning of those entries is.
Looking at all the other tables, the first entry in `SHORT_OFFSET_RUNS` does not match the lower bound, so it might just be coincidence.

Maybe its possible to do a `checked_sub` and take advantage of this, but I would have to dig deeper into the code.